modules/support_ticket/src/PageCache/DenySupportTicketPreview.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\support_ticket\PageCache\DenySupportTicketPreview.
+ */
+
+namespace Drupal\support_ticket\PageCache;
+
+use Drupal\Core\PageCache\ResponsePolicyInterface;
+use Drupal\Core\Routing\RouteMatchInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Cache policy for support ticket preview page.
+ *
+ * This policy rule denies caching of responses generated by the
+ * entity.support_ticket.preview route.
```

Comment:
no_cache: TRUE seems the most explicit to me -- updated the code here, will submit a patch for core (and check to see if this pattern occurs in other modules too).